polkadot-host-spec/ae-hostapi.tm

```diff
@@ -255,9 +255,21 @@
     containing the next key in lexicographic order.
   </itemize>
 
-  <section|Child Storage>
+  <section|Child Storage><label|sect-child-storages>
 
-  Interface for accessing the child storage from within the runtime.
+  Interface for accessing the child storage from within the runtime. Child
+  storages work the same way that regular storages do, but use two keys for
+  each value instead of just one. This allows for the functionality of having
+  identical keys with different values based on certain contextes, such as
+  separate storages for each Smart Contract and or Parachain, identified by
+  the preceding key. In case of Smart Contracts, the identifier could be an
+  address while for Parachain it could be the Parachain ID.
+
+  \;
+
+  Child storages are more performant than simply using prefixes, since the
```

Comment:
It s great that we have this comparison here but I think we can have it bit deeper. 
1) we can probably have example of a sub trie represented as prefix trie as well as a child trie and see how these two are diffenent.